packages/openai-base/src/tools/computer-use-tool.ts

```
@@ -28,11 +28,18 @@ export function convertComputerUseToolToAdapterFormat(
+      metadata: {
+        ...toolData,
+      },
     },
-  }
+    'computer_use',
+  )
 }
+import {
+  getOpenAIProviderToolMetadata,
+  openAIProviderTool,
+} from './openai-provider-tool'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)